modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/colorful-head/index.css

```
@@ -0,0 +1,8 @@
+.fragment_238 {
+	font-weight:var(--font-weight-bolder);
+	font-size: var(--display4-size);
+}
+
+.fragment_238 #color-text{
```

```suggestion
.colorful-head #color-text {
```

Spacing missing before `{`